com.github.utsushi.Utsushi.json

```diff
@@ -34,69 +34,10 @@
 				"./b2 install"
             ]
 		},
-        "shared-modules/gudev/gudev.json",
+                "shared-modules/gudev/gudev.json",
 		"shared-modules/libusb/libusb.json",
```

Comment:
indentation error. seems like your editor is misconfigured.

Reply:
Yea thanks.